src/backend/distributed/sql/udfs/create_time_partitions/latest.sql

```diff
@@ -0,0 +1,44 @@
+CREATE OR REPLACE FUNCTION pg_catalog.create_time_partitions(
+    table_name regclass,
+    to_date timestamptz,
```

Comment:
Open question: Is it helpful to specify a default here?

Reply:
I think default value can be provided via upper abstraction layer while we are implementing automated partition management. Having a default value for that function looks confusing to me.